src/webui/www/public/newlabel.html

```diff
@@ -7,6 +7,17 @@
     <script type="text/javascript" src="scripts/mootools-1.2-core-yc.js" charset="utf-8"></script>
     <script type="text/javascript" src="scripts/mootools-1.2-more.js" charset="utf-8"></script>
     <script type="text/javascript">
+        var newLabelKeyboardEvents = new Keyboard({
```

Comment:
This way is new for me and looks more complex than the old way => http://stackoverflow.com/a/7060762
Does this way work in old browsers likeIE8?